packages/app/src/components/SearchResultsCard/style.css

```diff
@@ -13,6 +13,28 @@
   background-color: white;
 }
 
+.search-result-span {
```

Comment:
Guessing these are from a different branch? We should remove them if they aren't used.